aepp/configs.py

```diff
@@ -367,3 +370,141 @@ def setSandbox(self,sandbox:str=None)->dict:
         self.header["x-sandbox-name"] = sandbox
         self.__configObject__["sandbox"] = sandbox
         return self.getConfigObject()
+
+    def setOauthV2setup(self,credentialId:str=None,orgDevId:str=None)->bool:
+        """
+        set the credential ID and the OrgIdDev as attributes of the instance.
+        * credentialId
+        * orgDevId
+        Returns True when success.
+        Argument:
+            credentialId : OPTIONAL : The credential id that can be found on your Project page.
+            orgDevId : OPTIONAL : the org Id but NOT the IMS string. It is defined on your project page.
+                Example : https://developer.adobe.com/console/projects/<orgId>/<projectId>/credentials/<credentialId>/details/oauthservertoserver
+        """
+        if self.connectionType != "oauthV2":
+            raise Exception("You are trying to set credential ID or orgDevId for auth that is not OauthV2. We do not support these auth type.")
+        if credentialId is None:
+            raise ValueError("credentialId is None")
+        if orgDevId is None:
+            raise ValueError("orgDevId is None")
+        self.credentialId = credentialId
+        self.orgDevId = orgDevId
+        return True
```

Comment:
why are we returning boolean from a mutator function, typically a mutator would return void and an accessor function would return a datatype

Reply:
True, for some reason, I thought it would be interesting to get when the function has correctly executed but definitely not normal setup, I will change it in the next commit.